app/frontend/wallet/types.ts

```diff
@@ -0,0 +1,12 @@
+import {NetworkId, ProtocolMagic} from './constants'
```

Comment:
I'd keep the NetworkId and ProtocolMagic types here together with other enums in constants to separate true constans from typing-related abstractions. This would also get us rid of the current cyclic import between types.ts ans constants.ts

Reply:
Will do, I was having a dilemma myself whether to define enums in `constants` file or `types` file. I agree let's have enums in `constants` file since as you said, they are typing-related abstractions and also remove the cyclic import. :+1: Thnx.